server/util/cosmos.js

```diff
@@ -20,7 +20,7 @@ import { PUBSUB_TOPIC_MISC } from '../constant';
 import publisher from './gcloudPub';
 
 import {
-  COSMOS_ENDPOINT as cosmosRPCEndpoint,
+  COSMOS_RPC_ENDPOINT as cosmosRPCEndpoint,
```

Comment:
```suggestion
  COSMOS_RPC_ENDPOINT,
```
Maybe could skip the rename step and directly use `COSMOS_RPC_ENDPOINT`  just like `COSMOS_GAS_PRICE` and other config parameters?